decidim-proposals/config/locales/hu.yml

```diff
@@ -103,6 +103,7 @@ hu:
         name: Javaslatok
         settings:
           global:
+            allow_card_image: Kártyakép engedélyezése
```

Comment:
As Crowdin project manages other languages translations, could you remove this hu.yml file please ?